quiche/src/recovery/mod.rs

```diff
@@ -185,6 +185,7 @@ pub trait RecoveryOps {
 
     fn pto(&self) -> Duration;
 
+    /// The most recent data delivery rate estimate in bytes/s.
     fn delivery_rate(&self) -> u64;
```

Comment:
Would be good to change this function to return a Bandwidth

fn delivery_rate(&self) -> Bandwidth;

Reply:
Address in https://github.com/cloudflare/quiche/pull/2043/commits/656aa246cb503d227f127332f143f4525352a935